Makefile

```diff
@@ -19,6 +19,16 @@ help:
 clean:
 	rm -rf *.egg-info
 
+publish:
```

Comment:
I can't comment on regions you didn't change, but a bit higher in the file than here, I would not mind seeing build do `make configure`.

Reply:
Done